CONTRIBUTORS.md

```diff
@@ -82,3 +82,8 @@ repository.
 
   * CC BY-SA 4.0 (by nixos-artwork)
       * `wallpapers/nix-wallpaper-dracula.png`
+
+## @papojari
+
+  * [Apache License 2.0](https://github.com/google/material-design-icons/blob/master/LICENSE)
```

Comment:
Is there a reason that you link to an external repository? This might change over time.
I think you should link to the official license http://www.apache.org/licenses/LICENSE-2.0

Reply:
[done](https://github.com/NixOS/nixos-artwork/pull/66/commits)